### DISCUSSION TITLE
Newbie Help

### DISCUSSION THREAD

**Discussion by BobbyPr** [missionpinball/missionpinball#104] [Q&A] [answered]
just trying to open  a simple config file and get this message.
Any ideas?
[2023-07-16-10-15-53-mpf-DESKTOP-8DSP8H2.log](https://github.com/missionpinball/missionpinball/files/12064392/2023-07-16-10-15-53-mpf-DESKTOP-8DSP8H2.log)

**Comment by toomanybrians** [comment, marked as answer]
This line `serial.serialutil.SerialException: could not open port '/dev/ttyACM0': FileNotFoundError(2, 'The system cannot find the path specified.', None, 3)` indicates that it's trying to open a serial port called `/dev/ttyACM0` in order to connect to some hardware, but that port is not valid.

I see from your log that you're running on Windows, and on Windows, port names are like COM4, COM5, etc. The /dev/... port name is a mac or linux format.

You need to change your config to use the ports on your actual computer. Or, if you do not have physical hardware connected, update the config, or run MPF in the mode (with -x or -X in the command line) so it doesn't try to connect to the hardware.

**Reply by BobbyPr** [reply]
thank you!!! Oh this is going to be a long haul .... lol
